docs/comit-protocol/comit-projects.md

```diff
@@ -0,0 +1,32 @@
+---
+id: comit-projects
+title: COMIT projects
+sidebar_label: COMIT projects
+---
+
+⚠️ Disclaimer: This part of the documentation is still under construction!
+
+This section of the documentation provides an overview of the different projects of the COMIT protocol.
+Sub-sections provide an introduction into project-specific details.
+
+The COMIT protocol is completely open source and consists of several repositories on GitHub.
+Here is a list of the most important repositories on GitHub:
+
+* [comit-rs](https://github.com/comit-network/comit-rs): The comit-rs project is the reference implementation of the protocol written in Rust. The [comit network daemon (cnd)](comit-protocol-stack.md#comit-network-daemon-cnd) is the binary that is released by the comit-rs project.
+* [comit-js-sdk](https://github.com/comit-network/comit-js-sdk): The COMIT Javascript SDK (comit-js-sdk) wraps the communication with cnd and provides an entry point for integrating atomic swaps with COMIT into Javascript applications. The comit-js-sdk is written Typescript.
+* [create-comit-app](https://github.com/comit-network/create-comit-app): The create-comit-app project includes Typescript demos and examples that show how to use the comit-js-sdk. Additionally it includes the `start-env` which spins up a development environment using docker containers.
+* [blockchain-contracts](https://github.com/comit-network/blockchain-contracts): The blockchain-contracts project contains the HTLC code that is used for atomic swaps in comit-rs.
```

Comment:
```suggestion
* [blockchain-contracts](https://github.com/comit-network/blockchain-contracts): The blockchain-contracts project contains the blockchain smart contracts that is used for atomic swaps in comit-rs.
```
It is likely to include more than HTLCs one day

Reply:
I don't want to use `smart contracts` because that sounds like it is limited to Ethereum. 
Can we just go for `blockchain contracts`?

Reply:
You can also go for `program` so it's a bit more agnostic of the underlying blockchain. Yet, blockchain money is `programmable money`.